webview/src/experiments/components/table/styles.module.scss

```diff
@@ -309,31 +309,37 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
       }
     }
 
-    &:not(.rowSelected) {
-      & > *:first-child {
-        position: sticky;
-        left: 0;
-        z-index: 3;
+    > *:first-child {
+      position: sticky;
+      left: 0;
+      z-index: 3;
 
-        &:after {
-          content: '';
-          height: 100%;
-          position: absolute;
-          top: 0;
-          width: 6px;
-          right: 0;
-          transition: box-shadow 0.25s;
-        }
+      &:after {
+        content: '';
+        height: 100%;
+        position: absolute;
+        top: 0;
+        width: 6px;
+        right: 0;
+        transition: box-shadow 0.25s;
       }
     }
 
     &.rowSelected {
       background-color: $row-bg-selected-color;
+
+      > *:first-child {
+        background-color: $row-bg-color;
+
+        &:before {
```

Comment:
I'm assuming the reason we chose to make selected rows not sticky originally was that the selected rows background color was partially transparent, leading to something like this if you don't use `:before` elements:

<img width="446" alt="image" src="https://user-images.githubusercontent.com/43496356/187464955-e2a53336-a36f-4830-b4cf-8da6be9663e6.png">